sonar-project.properties

```diff
@@ -2,7 +2,7 @@ sonar.projectKey=ionos-cloud_cluster-api-provider-proxmox
 sonar.organization=ionos-cloud
 
 sonar.sources=.
-sonar.exclusions=**/*_test.go,**/mock_*.go,**/zz_generated*.go,templates/crs/cni/**
+sonar.exclusions=**/*_test.go,**/mock_*.go,**/zz_generated*.go,templates/crs/cni/**,test/e2e/
```

Comment:
why?

Reply:
this would include the testing into coverage, 
we don't want that.

Reply:
tests for tests! :)